website/docs/d/service_account.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_service_account"
+sidebar_current: "docs-vcd-datasource-service-account"
+description: |-
+  Provides a data source to read VCD Service Accounts.
+---
+
+# vcd\_service\_account
+
+Provides a data source to read VCD Service Accounts.
+
```

Comment:
You could possibly add the same line here https://github.com/vmware/terraform-provider-vcd/pull/1070/files#diff-7d95f33c31e2ed51388acd78d47192e544df1c5c7a2e56a290b12fd3c89fe41eR13

Reply:
Added.